ShareTheTable/View Controllers/Profile Making/Question/QuestionViewController.h

```diff
@@ -0,0 +1,32 @@
+//
+//  QuestionViewController.h
+//  ShareTheTable
+//
+//  Created by Andrea Gonzalez on 7/7/22.
+//
+
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface QuestionViewController : UIViewController
+
+@property (weak, nonatomic) IBOutlet UIButton *otherFoodButton;
```

Comment:
Same thing as before, instead of 12 outlets and a lot of UI fixing in the storyboard, it would make it easier to use a CollectionView. This would also help with the issue you mentioned in your summary about having 12 methods for each button action - you could use collectionView methods to do things in one place.